packages/twenty-front/src/modules/ui/layout/tab/hooks/useTabList.ts

```diff
@@ -0,0 +1,18 @@
+import { useTabListStates } from '@/ui/layout/tab/hooks/internal/useTabListStates';
+import { useRecoilState } from 'recoil';
+
+
+export const useTabList = (tabListId?: string) => {
+  const {
+    activeTabIdState,
+  } = useTabListStates({
+    tabListScopeId: `${tabListId}-scope`,
+  });
+
+  const [activeTabId, setActiveTabId] = useRecoilState(activeTabIdState);
```

Comment:
remove activeTabIdStateScopeMap
introduce isTabActiveFamilyStateScopeMap

add getFamilyState in useTabListStates

in useTableList:
return isTabActiveFamilyState